UnitTests/GitUI.Tests/Theming/AppColorDefaultsTests.cs

```diff
@@ -9,25 +12,43 @@ namespace GitUITests.Theming
     [TestFixture]
     public class AppColorDefaultsTests
     {
+        private string _originalPath;
+
+        [OneTimeSetUp]
+        public void Setup()
```

Comment:
Since this is a fixture setup, I propose we call it `OneTimeSetUp`?
`Setup` is typically decorated with `SetUpAttribute`, and it is run before each test.

Reply:
ok